wasm/Cargo.lock

```diff
@@ -1,30 +1,159 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+[[package]]
```

Comment:
We should be able to delete this whole file now that this package is part of the workspaces.